docs/supported_constellations.rst

```diff
@@ -0,0 +1,18 @@
+Supported Constellations
+=====================
+
+DE2
+---
+
+The Dynamics Explorer 2 spacecraft
```

Comment:
Plesae don't hate me.

I think it would be a good idea to provide links here to the relevant supported instruments.

🦆 <- me ducking out of the way.